WalletWasabi.Gui/Converters/MoneyBrushConverter.cs

```diff
@@ -20,6 +20,11 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 				var money = decimal.Parse(moneyString);
 				return money < 0 ? Brushes.IndianRed : Brushes.MediumSeaGreen;
 			}
+			else if (value is Money inMoney)
```

Comment:
Remove code duplication by adding logic. How about parsing everything to decimal and at the end return the color?